backend/src/achievements/entities/achievements.entity.ts

```diff
@@ -0,0 +1,20 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+export class Achievement {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({ unique: true, nullable: false })
+  name: string;
+
+  @Column({ unique: true, nullable: false })
+  description: string;
+
+  @Column({
+    type: 'bytea',
+    nullable: true,
+    default: null,
+  })
+  icon: Uint8Array;
```

Comment:
Did you consider storing the images as static files and just storing the path of the icon? As the `achievement` icons are not user content and won't be modified, I think we can store them as static files, which may improve performance and database size. But that doesn't mean we have to do so

Reply:
Makes sense, I'll change it !

Reply:
For the record, we -tmorris and I- ended up deciding to keep it that way until we have a clearer overall idea of:
-how the achievements will be populated
-how they will be displayed by the frontend.